adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -133,6 +136,7 @@ private void parsingNoticeToProto(String filename, int lineNumber, String fieldN
                 .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.ERROR)
                 .setCsvColumnName(lineNumber, fieldName)
                 .setEntityValue(rawValue)
+                .clearAltEntityId()
```

Comment:
Good catch! As we said together, replace this by a call to `clear()` at the beginning of the chain to clear the reused internal builder